src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -199,6 +199,16 @@ bool LibraryCallKit::arch_supports_vector(int sopc, int num_elem, BasicType type
         tty->print_cr("  ** Rejected vector op (%s,%s,%d) because architecture does not support variable vector shifts",
                       NodeClassNames[sopc], type2name(type), num_elem);
       }
+#endif
+      return false;
+    }
+  } else if (VectorNode::is_integer_negate(sopc)) {
+    if (!VectorNode::is_vector_negate_supported(sopc, num_elem, type, false)) {
+#ifndef PRODUCT
+      if (C->print_intrinsics()) {
+        tty->print_cr("  ** Rejected vector op (%s,%s,%d) because architecture does not support variable vector negate",
```

Comment:
"variable vector negate" seems a bit strange to me.
How about removing "variable"?